### DISCUSSION TITLE
Request for Compatible jQuery File for Old Version Project

### DISCUSSION THREAD

**Discussion by xielinhan** [magicbell/community#155] [General]
Our product's front-end is built on the JSF framework, and we are using an older version of jQuery for compatibility reasons.

Project Background: Our product's front-end is built on the JSF framework, and we are using an older version of jQuery for compatibility reasons.

Requirement: We are seeking a jQuery file similar to jquery.min.js that is compatible with our older version jQuery project. Our current jQuery file version is jquery-1.4.4.min.js.

Detailed Description: We aim to seamlessly integrate a notification bell component into our product's front-end, and the current requirement is to obtain a jQuery file that allows proper functionality.

**Comment by unamashana** [comment]
Unfortunately, we are unable to support older versions.
